src/System.Buffers/src/System.Buffers.csproj

```diff
@@ -1,17 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <AssemblyVersion>4.0.3.0</AssemblyVersion>
```

Comment:
I had a question about this which I wrote in my Bcl.HashCode PR: Why is the AssemblyVersion 4.0.3.0 when the largest number I saw in 3.1 was 4.0.5.0?

Reply:
Maybe this is a question for @ericstj.

Reply:
Where do you see a 4.0.5.0 assembly version for System.Buffers? The latest available package on nuget.org has 4.0.3.0.

Reply:
@carlossanlop - in 3.1 this package [was already gone](https://github.com/dotnet/corefx/tree/release/3.1/src/System.Buffers) - so the version there isn't the one we need to follow.

2.1 was [the last place this shipped](https://github.com/dotnet/corefx/tree/release/2.1/src/System.Buffers)

We should consider making AssemblyVersioning easier in this repo if that's possible.  Make it as easy as it is in the latest 6.0+ branches where we just ask folks to increment a single number without thinking about the versioning rules.

The constraints are that the version needs to be _higher_ than the last shipped version, and _lower_ than any place where we expect a framework version to win over the package.